### PR TITLE
MODLD-507: Review and cleanup Module Descriptor for mod-linked-data

### PR DESCRIPTION
**Pull request by askhat-abishev**

### CODE REVIEW

File: descriptors/ModuleDescriptor-template.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
         {
           "methods": [ "GET" ],
           "pathPattern": "/resource/{id}",
-          "permissionsRequired": [ "linked-data.resources.bib.get" ],
+          "permissionsRequired": [ "linked-data.resources.bib.item.get" ],
```
[the remark this answers is not in the document]
Comment (askhat-abishev) [reply descriptors/ModuleDescriptor-template.json:38]: I've named it according to the following convention https://folio-org.atlassian.net/wiki/spaces/FOLIJET/pages/156368925/Permissions+naming+convention#3.-Entity-Scope

[the remark this answers is not in the document]
Comment (PBobylev) [reply descriptors/ModuleDescriptor-template.json:38]: alright then!

**Review by PBobylev — APPROVED** [review]

File: descriptors/ModuleDescriptor-template.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (askhat-abishev) [reply descriptors/ModuleDescriptor-template.json:38]: right

[the remark this answers is not in the document]
Comment (askhat-abishev) [reply descriptors/ModuleDescriptor-template.json]: done

File: descriptors/ModuleDescriptor-template.json [hunk 2]
```diff
@@ -32,12 +32,6 @@
           "permissionsRequired": [ "linked-data.resources.bib.get" ],
           "modulePermissions": []
         },
-        {
-          "methods": [ "GET" ],
-          "pathPattern": "/resource/{id}",
```
Comment (PBobylev) [inline descriptors/ModuleDescriptor-template.json:37]: not this one :) it's used by the UI. 
`"pathPattern": "/resource"` (line 31) isn't used and should be deleted

**Review by PBobylev — CHANGES_REQUESTED** [review]

File: descriptors/ModuleDescriptor-template.json (hunk 2 above)
Comment (askhat-abishev) [reply descriptors/ModuleDescriptor-template.json:37]: oh, indeed
fixed, thanks!

**Review by PBobylev — APPROVED** [review]

File: descriptors/ModuleDescriptor-template.json [hunk 3]
```diff
@@ -134,8 +128,8 @@
   "permissionSets": [
     {
       "permissionName": "linked-data.resources.bib.get",
-      "displayName": "Linked Data: Get a bibliographic (Work or Instance) resource",
-      "description": "Get a bibliographic (Work or Instance) linked-data resource"
+      "displayName": "Linked Data: Get a bibliographic (Work or Instance) resources",
+      "description": "Get a bibliographic (Work or Instance) linked-data resources"
```
Comment (PBobylev) [inline descriptors/ModuleDescriptor-template.json:132]: no no, we should keep singular form, right?

**Review by pkjacob — APPROVED** [review]

**Pull request merged**